client/src/views/NewProject.vue

```diff
@@ -1,40 +1,43 @@
 <template>
   <html>
-    <section id="new" class="main">
+    <head>
+      <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&family=Comfortaa:wght@700&family=Lato&family=Patrick+Hand&display=swap"
+        rel="stylesheet"
+      />
+    </head>
+    <section class="main">
       <form>
         <h1 class="title is-1">
           New Project
         </h1>
         <div class="field">
           <label class="label">Project Name:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.projectName" />
+            <input class="input" type="text" size = "40" v-model="form.projectName" />
           </div>
         </div>
         <div class="field">
           <label class="label">Description:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.description" />
+            <textarea rows="4" cols="50" v-model="form.description"></textarea>
           </div>
         </div>
         <div class="field">
           <label class="label">Links:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.links" />
+            <input class="input" type="text" size = "75" v-model="form.links" />
```

Comment:
How is the user meant to addd multiple links? Comma separated list? One at a time?

Reply:
I think we could honestly do multiple with spaces or comma & space (just need to separate links and create an array).